manifest.json

```diff
@@ -375,6 +375,32 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm",
+              "dockerfile": "src/ubuntu/16.04/helix/arm32v7",
```

Comment:
Following the alpine convention the path would be:

src/ubuntu/16.04/arm32v7/helix/

Reply:
I followed alpine 3.6 convention src/alpine/3.6/ForHelix/amd64

@MichaelSimons, can you comment on what is the expected convention?

Reply:
On closer look this looks correct thanks!